fabfile.py

```diff
@@ -88,17 +82,17 @@ def get_db_dump(dbname, clean=True):
 
     """
     require('environment')
-    if not files.exists("%(home)s/.pgpass" % env):
-        abort("Please get a copy of .pgpass and put it in your home dir on the server of interest (not your local system)")
+    run('sudo -u pycon /srv/pycon/env/bin/python /srv/pycon/pycon/manage.py sqldsn -q -s pgpass -R default 2>/dev/null > ~/.pgpass')
+    run('chmod 600 ~/.pgpass')
     dump_file = '%(project)s-%(environment)s.sql' % env
     flags = '-Ox'
+    dsn = sudo('/srv/pycon/env/bin/python /srv/pycon/pycon/manage.py sqldsn -q -R default 2>/dev/null', user='pycon').stdout
```

Comment:
Oh I see. (Feeble excuse: these long lines are hard to review in GitHub's diff viewer, which seems to think anything beyond a certain width is not worth seeing.)